backend/src/email/utils/callback/parsers/sendgrid.ts

```diff
@@ -29,19 +29,14 @@ type SendgridRecord = {
   'smtp-id': string
   timestamp: number
 }
-
-function stringifyPayload(payload: SendgridEvent): string {
-  return `[${payload.map((event) => JSON.stringify(event)).join(',\r\n')}]\r\n`
-}
-
 const isEvent = (req: Request): boolean => {
   const signature = req.get(SIGNATURE_HEADER)
   const timestamp = req.get(TIMESTAMP_HEADER)
   if (!(signature && timestamp && req.body)) {
     return false
   }
   const decodedSignature = Signature.fromBase64(signature)
-  const timestampPayload = timestamp + stringifyPayload(req.body)
+  const timestampPayload = timestamp + req.body
```

Comment:
The JSON body-parser causes `req.body` to be a JSON object, but we need it to be the `string` that was sent in the callback request in its original form, including any `\r\n` characters.

Reply:
Here's an example from the official SendGrid samples: https://github.com/sendgrid/sendgrid-nodejs/commit/8110b173007049836455198450947780e9a70416